### ISSUE TITLE
Timesheet Module >Calendar && favorites>Secondary header

### ISSUE THREAD

**Issue by cfmitrah** [cfmitrah/entek_public_issues#12]
Browser: Google Chrome Version 87.0.4280.66 (Official Build) (64-bit)
staging Version : 7.0.9

React_Site Url :http://219.89.201.81:3397/react/

- [ ] Table size is fully shown in the existing for the expense table whre as in our host it is halfly shown; **-- Low priority**

- [x] When we opened the favorites then switch to the new resource ,then switch to project  or any module then return back to timesheet module  in that manage resource icon was enabled in our host;

![manage resource show issue ](https://user-images.githubusercontent.com/532401/100596028-afb8e700-3321-11eb-9672-1ce514fa5767.gif)

- [x] Similar to the Above ,the transaction  details also not shown in the same scenario it needs to click the manage resource icon to show the transaction details

![save as issue ](https://user-images.githubusercontent.com/532401/100596426-2950d500-3322-11eb-9523-f40157afca72.gif)

- [x] While opening the favorites the selected  client details are not shown in host ,it works only for the task not for the client and  vice versa; 

![favourites client details](https://user-images.githubusercontent.com/532401/100596849-a54b1d00-3322-11eb-9ca2-47f255ac7d14.gif)

- [ ] undo and redo didn't work when we changing from the 5 day view to seven day view, if i made a any changes in the 5 day view calendar then i have change to seven day view then  again if i back to 5 day view i was not able to undo or redo the changes i have made; **-- need to check whether it's acceptable or not.**
![undoredo](https://user-images.githubusercontent.com/532401/100599286-d24cff00-3325-11eb-8869-5559aa2f2887.gif)

- [x] if we add a new task to the favorites it didn't get added only after adding another task only it shows the two values,similar to the delete also
after refreshing the page only it 'll the show the added client details in the favorites ==>Moved to Ticket #204 

![api data not loading](https://user-images.githubusercontent.com/532401/100599459-12ac7d00-3326-11eb-8629-4388d72be783.gif)

- [x] While searching it didn't filter the search values when we change the value through View by --this is from  V1_Time Sheet Calendar Issue #163 (Staging Version 7.0.8:)

![image](https://user-images.githubusercontent.com/532401/99868649-5696d800-2bea-11eb-8e4f-4f419f598266.png)

- [x] While switching on the resource, if we tend to add a favorites the data didn't get added instead it shows the login resource favorites in the new resource && it is similar to the delete also
